src/app/legacy/containers/StoryPromo/__snapshots__/index.test.jsx.snap

```diff
@@ -3471,9 +3471,9 @@ exports[`StoryPromo Container should render featureLink correctly for amp 1`] =
 }
 
 .emotion-6 {
-  height: 0;
   overflow: hidden;
   position: relative;
+  height: 0;
```

Comment:
These snapshot changes just seem to be inocuous reorders, bit of a pain but looks okay

Reply:
Yep most of them are just that annoyingly